test/utils/wabt_engine.cpp

```diff
@@ -37,123 +49,126 @@ std::unique_ptr<WasmEngine> create_wabt_engine()
 
 bool WabtEngine::parse(bytes_view input) const
 {
-    wabt::interp::Environment env;
-
-    wabt::interp::HostModule* hostModule = env.AppendHostModule("env");
-    assert(hostModule != nullptr);
-
-    hostModule->AppendFuncExport("adler32", {{wabt::Type::I32, wabt::Type::I32}, {wabt::Type::I32}},
```

Comment:
Nice, it doesn't require this for parsing anymore?

Reply:
Yes, parsing is now a normal separate step.